qdev_wrappers/fitting/AnalysisToSQL.py

```diff
@@ -0,0 +1,151 @@
+
+
+import sqlite3
+import dill
+from qdev_wrappers.fitting.Fitclasses import T1, T2
+
+
+def is_table(tablename, cursor):  # Checks if table already exists. Assumes database connection and cursor already established
+
+    table_count = "SELECT count(*) FROM sqlite_master WHERE type = 'table' AND name='{}'".format(tablename)
+    execute = cursor.execute(table_count)
+    count = execute.fetchone()[0]
+
+    if count == 0:
+        return False
+
+    if count == 1:
```

Comment:
if the `count` is not 0 or 1 the function does not return anything. It would be great to throw an error in this situation.